x-pack/plugins/ml/server/models/data_recognizer/modules/apm_transaction/manifest.json

```diff
@@ -1,29 +1,29 @@
 {
   "id": "apm_transaction",
   "title": "APM",
-  "description": "Detect anomalies in transactions from your APM services.",
+  "description": "Detect anomalies in transactions from your APM services for metric data.",
   "type": "Transaction data",
   "logoFile": "logo.json",
-  "defaultIndexPattern": "apm-*-transaction",
+  "defaultIndexPattern": "apm-*-metric,metrics-apm*",
   "query": {
     "bool": {
       "filter": [
-        { "term": { "processor.event": "transaction" } },
-        { "exists": { "field": "transaction.duration" } }
+        { "term": { "processor.event": "metric" } },
+        { "term": { "metricset.name": "transaction" } }
```

Comment:
The manifest query will need to be
```
"query": {
    "bool": {
      "filter": [
        { "term": { "processor.event": "metric" } },
        { "term": { "metricset.name": "transaction" } }
      ],
      "must_not": { "terms": { "_tier": [ "data_frozen", "data_cold" ] } }
    }
  },
```

but as discussed, feel free to leave the `_tier` part out of the changes in this PR and I can include them when editing all the other ML modules as part of https://github.com/elastic/kibana/issues/116696.